tests/phpunit/integration/tests/Story_Post_Type.php

```diff
@@ -66,79 +73,77 @@ public static function wpSetUpBeforeClass( $factory ) {
 		set_post_thumbnail( self::$story_id, $poster_attachment_id );
 	}
 
-	public function setUp() {
-		parent::setUp();
+	public function set_up() {
+		parent::set_up();
+
+		$this->instance = new \Google\Web_Stories\Story_Post_Type();
```

Comment:
Again, it is a bad idea to reuse objects like this. New test new instance.

Reply:
Keeping dicussion in one place at https://github.com/google/web-stories-wp/pull/8607#discussion_r716849381